content/blogs/YYYY-MM-DD-CI-CD-kestra-comparison.md

```diff
@@ -0,0 +1,186 @@
+By now, you’re surely seeing the pattern: the tools we mentioned are great for CI/CD and managing simple workflows, while Kestra is purpose-built for orchestrating complex workflows across different systems. So, basically, if you're working on the GitHub environment and need a CI solution, you'll use GitHub Actions, and the same thing applies if you're in the GitLab ecosystem.
+
+But what if you need to manage workflows across multiple environments? Here's where Kestra comes into the game!
+
+So, let’s zero in on a few of Kestra’s key features that make it stand out:
+
+- **Tasks Orchestration**: Kestra’s primary strength is in orchestrating tasks across multiple environments. It doesn’t matter if those tasks are happening on different cloud providers, on-prem servers, or in a hybrid environment: Kestra manages the flow of tasks smoothly. For example: CI/CD tools can kick off a build or deploy code, but Kestra can handle workflows that include those steps plus data processing tasks, external API calls, or anything else your workflow requires.
+
+![Workflow orchestration in Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_orchestration.jpg)
+
+- **Error Handling and Retries**: In traditional automation tools, error handling and retries often require custom scripting. Kestra, instead, bakes this into its core. In fact, if a task fails, Kestra can automatically retry it, adjust workflows dynamically, or trigger error-specific branches in the workflow. This is a huge time-saver when working with workflows prone to errors, like distributed systems or long-running data processing jobs.
+
+![Managing errors in Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_errors.jpg)
+
+- **Cloud-Native and Distributed Systems Support**: Kestra was built with modern [cloud-native](./2022-06-21-light-architecture.md) and distributed systems in mind. So, whether you’re orchestrating workflows across AWS, GCP, Azure, or on-prem services, Kestra can handle it all, as it’s designed on purpose to manage tasks that live in various environments without extra configuration or complexity.
+
+![Plugins for Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_plugins.jpg)
+
+- **Event-Driven Architecture**: A lot of modern workflows are event-driven, and Kestra really shines here. You can trigger workflows based on events happening across your stack, whether that’s a file being uploaded, a database change, or an API call: this makes Kestra super flexible for managing real-time and batch workflows alike.
+
+![Real time triggers in Kestra](/blogs/2024-06-27-realtime-triggers.jpg)
+
+- **Visual Workflow Editor**: While CI/CD tools often require you to dive into script configurations, Kestra offers a visual workflow editor. In fact, you can build and manage your workflows in a more intuitive, visual way, reducing the cognitive load of managing complex systems.
+
+![Workflows in Kestra](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/kestra_flows.jpg)

```

Comment:
Can you use a different image over a thumbnail. Confusing as these look like the video. If you mean to put the video, then put that. I can make a similar image to the thumbnail if that's what you're after